README.rst

```diff
@@ -14,51 +14,59 @@ backports.cached_property
 .. image:: https://img.shields.io/badge/code%20style-black-000000.svg
     :target: https://github.com/ambv/black
 
-What
-----
+Introduction
+------------
 
-Python 3.8 adds great descriptor to functools: cached_property.
-Technically all required APIs was available since python 3.6,
-but it is what it is.
+Python 3.8 adds a great decorator to functools: ``cached_property``.
+Technically all required APIs were available since Python 3.6, but
+releases until 3.8 don't have it and users need to live with this reality.
 
-This package is a backport of this functionality for python 3.6 and 3.7.
+Therefore, this package backports said feature to python 3.6 and 3.7.
 
 How to use
 ----------
 
+Simply do
+
 .. code-block:: python
 
     from backports.cached_property import cached_property
 
-And then python 3.8 documentation will work (because code is minimally changed):
+and it will work as documented upstream (see copy below).
 
-.. class:: cached_property
```

Comment:
Actually, I think this was syntactically correct, only GitHub's renderer doesn't like it (e.g. the `.. class:: cached_property` header was not displayed at all).

I changed it to make it play with GH, though this is a displeasing situation.